src/ctapipe/image/cleaning.py

```diff
@@ -466,12 +466,36 @@ class ImageCleaner(TelescopeComponent):
     ``ImageCleaner.from_name()`` to construct an instance of a particular algorithm
     """
 
+    def __call__(self, tel_id: int, event: ArrayEventContainer) -> np.ndarray:
+        """
+        Calls the relevant functions to identify pixels with signal
+        and reject those with pure noise.
+
+        Parameters
+        ----------
+        tel_id: int
+            which telescope id in the subarray is being used (determines
+            which cut is used)
+        event: `ctapipe.containers.ArrayEventContainer`
+
+        Returns
+        -------
+        np.ndarray
+            boolean mask of pixels passing cleaning
+        """
+        mask = self.clean_image(
+            tel_id=tel_id,
+            image=event.dl1.tel[tel_id].image,
+            arrival_times=event.dl1.tel[tel_id].peak_time,
+        )
+        return mask
+
     @abstractmethod
```

Comment:
Not sure if this should now be an abstractmethod, since we explicitly changed the API to support more inputs. I.e. we shouldn't force people to implement the restricted API.

Reply:
If people not implement the low-level cleaning method, the `VolumeReducer` can't use it, can it?

Reply:
Yes.

Which is I think for the moment ok, we don't take anything away from `VolueReducer` that's currently supported.

In a future PR, we could update the VolumeReducer though to use the `__call__` API.